src/corehost/cli/args.cpp

```diff
@@ -53,6 +53,16 @@ void setup_shared_store_paths(const hostpolicy_init_t& init, const pal::string_t
         args->dotnet_shared_store = own_dir;
         append_path(&args->dotnet_shared_store, RUNTIME_STORE_DIRECTORY_NAME);
         append_path(&args->dotnet_shared_store, init.tfm.c_str());
+
+        //Also add an architecture specific path to allow for unzipped developer cli experience
+        //where developer points runtime store operation output to "dotnet.exe" relative shared store folder
+
+        pal::string_t local_developer_store = own_dir ;
+        append_path(&local_developer_store, RUNTIME_STORE_DIRECTORY_NAME);
+        append_path(&local_developer_store, get_arch());
+        append_path(&local_developer_store, init.tfm.c_str());
+        local_developer_store.push_back(PATH_SEPARATOR);
+        (args->env_shared_store).push_back(local_developer_store);
```

Comment:
Agreed about (1) above. It was not reflected in the change and hence, I was not sure how it was being planned - hence, my questions. To account for the global location to have the arch, I presume platform installers for the Store artifacts would need to carry the change, right? Also, looking up using Arch is specific to store location only.